package.json

```diff
@@ -19,7 +19,6 @@
         "jest": "^29.7.0"
     },
     "optionalDependencies": {
-        "ajv": "^8.12.0",
```

Comment:
is it not required to mention ajv as optional dependency ?

Reply:
two reasons to avoid this for now:
- there's one vulnerability with one of it's transitive dep.
- too many transitive dependencies

Unfortunately, npm install ---> installs with optional dependencies unleass we add extra option (i.e., --omit=optional).